### PR TITLE
add: env handler

### PR DESCRIPTION
**Pull request by TaTsuuYa**

### CODE REVIEW

File: executor.c [hunk 1]
```diff
@@ -21,6 +21,8 @@ int executor(char **args, char **env)
 		else
 			exit(0);
 	}
+	else if (_strcmp(args[0], "env"))
+		print_env(env);
```
Comment (imenyoo2) [inline executor.c]: u need to return or change the next `if` to `else if`

**Review by imenyoo2 — CHANGES_REQUESTED** [review]

File: executor.c (hunk 1 above)
Comment (imenyoo2) [reply executor.c]: return 0 to indicate success

File: built-ins.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include "main.h"
+
+/**
+ * handle_builtins - handles the built in commands
+ * @args: command
+ * @env: env vars
+ */
+
+void handle_builtins(char **args, char **env)
```
Comment (imenyoo2) [inline built-ins.c]: return int to indicate if `executor` should continue or not

File: built-ins.c [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+{
+	if (_strcmp(args[0], "exit"))
+	{
+		if (args[1] != NULL)
+			exit(_atoi(args[1]));
+		else
+			exit(0);
+	}
+
+	if (_strcmp(args[0], "env"))
```
Comment (imenyoo2) [inline built-ins.c:21]: probably should print an error if for example command is `env blabla`, the err msg is `env: 'blabla': No such file or directory`, u don't need to test for `blabla` if it's a valid file or dir, i tried it in my shell and whatever the second arg is it prints the same err

File: executor.c [hunk 4]
```diff
@@ -13,16 +13,12 @@ int executor(char **args, char **env)
 	int status;
 	char *newCommand;
 
-	/* handeling exit */
-	if (_strcmp(args[0], "exit"))
+	/* handeling built-ins */
+	if (_strcmp(args[0], "exit") || _strcmp(args[0], "env"))
```
Comment (imenyoo2) [inline executor.c]: `handle_builtins` already checks for `exit` and `env`, u just need to return 1 or 0 from `handle_builtns`

**Review by imenyoo2 — CHANGES_REQUESTED** [review]

**Comment by TaTsuuYa** [comment]
everything got implemented, should I merge

File: built-ins.c [hunk 5]
```diff
@@ -17,6 +19,18 @@ void handle_builtins(char **args, char **env)
 	}
 
 	if (_strcmp(args[0], "env"))
-		print_env(env);
+	{
+		if (args[1] != NULL)
+		{
+			write_std("env: '", STDERR_FILENO);
```
Comment (imenyoo2) [inline built-ins.c]: the `'` should be ` actually

**Review by imenyoo2 — CHANGES_REQUESTED** [review]

**Pull request merged**